packages/lwt-canceler/lwt-canceler.0.2/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" ]
+homepage: "https://gitlab.com/nomadic-labs/lwt-canceler"
+bug-reports: "https://gitlab.com/nomadic-labs/lwt-canceler/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/lwt-canceler.git"
+license: "MIT"
+depends: [
+  "ocaml"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
according to the dune-project file. Can you also check if lwt needs lower-bounds?

Reply:
I'm adding dune bounds (`1.7`) and fixing `dune-project` accordingly.

For `lwt` I found that the code was once used with `lwt 3.0.0` so I'm adding that as a bound.